pyproject.toml

```diff
@@ -19,7 +19,7 @@ documentation = "https://yapapi.readthedocs.io/en/stable/api.html"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.8.0"
```

Comment:
any specific reason? or is it just time?

Reply:
There was some specific reason (I don't remember now), probably not very important, but I also think it's time.